src/index.html

```diff
@@ -9,6 +10,33 @@
   <link rel="stylesheet" href="./style.css">
 </head>
 <body>
-  <h1>Search bar airbnb</h1>
+  <div>
+    <form data-qa="big">
+      <div class="box_big">
+        <label for="big-search">
```

Comment:
for attribute is not necessary here

Reply:
> ![image](https://user-images.githubusercontent.com/33497431/80190210-937efa00-861c-11ea-8b3f-3530affb30df.png)
> have you checked the demo link?
yes, it seems it works
https://prnt.sc/s53ca6

Reply:
https://prnt.sc/s53djp